src/node-file-trace.js

```diff
@@ -73,6 +74,9 @@ class Job {
     }
     this.base = base;
     this.cwd = resolve(processCwd || base);
+    if (!Array.isArray(exports) && typeof exports !== 'boolean')
+      exports = false;
+    this.exports = exports === true ? ['node'] : exports;
```

Comment:
I thought `exports: true` would mean all exports would get traced?

Reply:
We only need to trace the resolution for the target environment. Eg no need to trace a "browser" or "react-native" build if we aren't building for that platform.

Reply:
If that’s the case, the a better default is [node] instead of true and then the value can always be an Array.

Reply:
This package is called node-file-trace so we should assume node unless the user opts out

Reply:
The problem is turning it on by default will break Node.js 10 support.

Reply:
Is it possible for the consumer-side to know the Node.js version? If not an alternative would be to support both Node.js versions without and without "exports" at the cost of extra tracing.